src/main/java/org/jenkinsci/plugins/maven_artifact_choicelistprovider/AbstractRESTfulVersionReader.java

```diff
@@ -49,8 +48,7 @@ public AbstractRESTfulVersionReader(String pURL) {
 	}
 
 	void init() {
-		ClientConfig config = new ClientConfig();
-		Client client = ClientBuilder.newClient(config);
+		Client client = ClientBuilder.newClient();
```

Comment:
The new `ClientBuilder#newClient` is accepting a `Configuration` object, not a `ClientConfig`. I couldn't find any details about this migration. However, as the `ClientConfig` had nothing specific here, I believe this has the same behaviour.

Reply:
I did a similar migration in https://github.com/jenkinsci/urltrigger-plugin/pull/46. But this looks right to me.